_config.yml

```diff
@@ -10,7 +10,6 @@ github_username: OpenINF
 repository: OpenINF/open.inf.is
 
 # Site build settings.
-markdown: kramdown
```

Comment:
`markdown : kramdown` is the default, so we can omit it

Refs: https://jekyllrb.com/docs/configuration/default/